admin/tool/policy/lang/en/tool_policy.php

```diff
@@ -30,7 +30,10 @@
 $string['agelocationverification'] = 'Age and location verification';
 $string['ageofdigitalconsentmap'] = 'Age of digital consent';
 $string['ageofdigitalconsentmap_desc'] = 'Allows to define the age of digital consent. Users below the age are not allowed to accept the policy documents on their own. In such a case, policies must accepted by the holder of parental responsibility over the user. The age may be different in different countries. If there is no explicit value provided for the user\'s country code, the wildcard character\'s value is used.';
+$string['agreepolicies'] = 'Please agree to the following policies';
 $string['backtohome'] = 'Back to home';
+$string['backtotop'] = 'Back to top';
+$string['consentpagetitle'] = 'Policy Summary and Agreements';
```

Comment:
In Moodle default English pack, headings do not use "Capital Letters", only the first one should be capital